File: ZakaZaka/ClientApp/src/app/service/restaurant/IRestaurantReview.service.ts
import {RestaurantReview} from "../../Model/restaurantReview";

export interface IRestaurantReviewService{
  get(restaurantId: number) : RestaurantReview[];
  add(restaurantReview: RestaurantReview, restaurantId: number) : void;
  update(restaurantReview: RestaurantReview) : void;
  remove(reviewId: number) : void;
}

File: ZakaZaka/Service/RestaurantServices/RestaurantReview/IRestaurantReviewService.cs
using System.Collections.Generic;
using System.Threading.Tasks;
using ZakaZaka.Models.ModelsDTO;
using ZakaZaka.Models.Restaurants;

namespace ZakaZaka.Service.RestaurantServices
{
    public interface IRestaurantReviewService
    {
        public Task<IEnumerable<RestaurantReviewDTO>> Get(int restaurantId);

        public void Add(RestaurantReviewDTO restaurantReview);
        
        public void Update(RestaurantReviewDTO restaurantReview);
        
        public Task Remove(int reviewId);

        public Task SaveDataBase();
    }
}